kloppy/tests/test_statsbomb.py

```diff
@@ -68,6 +69,20 @@ def test_correct_deserialization(self):
             attacking_direction=AttackingDirection.NOT_SET,
         )
 
+        assert (
+            dataset.events[791].get_qualifier_value(BodyPartQualifier).value
+            == "HEAD"
+        )
+
+        assert (
+            dataset.events[2231].get_qualifier_value(BodyPartQualifier).value
```

Comment:
Please change this to `dataset.events[2231].get_qualifier_value(BodyPartQualifier) == BodyPart.RIGHT_FOOT`?

Reply:
Good idea. It's done